content/github/collaborating-with-issues-and-pull-requests/about-collaborative-development-models.md

```diff
@@ -12,7 +12,12 @@ topics:
   - pull requests
 ---
 
-There are two main types of development models with which you'd use pull requests. In the *fork and pull model*, anyone can fork an existing repository and push changes to their personal fork. You do not need permission to the source repository to push to a user-owned fork. The changes can be pulled into the source repository by the project maintainer. When you open a pull request proposing changes from your user-owned fork to a branch in the source (upstream) repository, you can allow anyone with push access to the upstream repository to make changes to your pull request.  This model is popular with open source projects as it reduces the amount of friction for new contributors and allows people to work independently without upfront coordination.
+There are two main types of collaborative development models with which you'd use pull requests.
+
+- Fork and pull model
```

Comment:
Thank so much for opening a PR @Baalkikhaal! You're right, this page is woefully out of date and could use some attention 💛 

Maybe instead of adding them as an unordered list manually, we can update the intro (in the front matter) to read:

> The way you use pull requests depends on the type of development model you use in your project. You can use the fork and pull model or the shared repository model.

Then you can remove this unordered list and create subheaders for the two sections instead. With subheadings, the table of contents will be autogenerated, just like it is for the rest of the docs.

While you're here, would you mind removing the italics around *fork and pull* and *shared repository model*? We don't use italics in our docs anymore.

Would you ping me once you've made these changes and I'll get this up for review?

Reply:
Thank you @janiceilene for replying to the pull request. Actually I am trying to wade through the waters of pull-requests cautiously. Honestly, I need some hand-holding from people like you😅. I will ping you once I understand how to update your recommended changes to the PR 👍?

Reply:
@janiceilene I have applied the changes that you have suggested. Please have a look into it 😄.